_data/projects/soundrecorder.yml

```diff
@@ -0,0 +1,15 @@
+name: SoundRecorder
+desc: A simple sound recording Android app implementing Material Design. The goal of this project is to develop as stable as possible sound recorder application.
+site: https://github.com/naXa777/SoundRecorder
```

Comment:
@naXa777 I can see that this is a fork, are you sure that you want to add a fork? you can also move your fork to a new repo if the owner of the repo doesn't have any problem.

Reply:
And what's wrong with forks? It's a fork of https://github.com/dkim0419/SoundRecorder which is licensed under [GNU v3](https://github.com/dkim0419/SoundRecorder/blob/master/LICENSE) so modifications and distribution are permitted.

Reply:
@naXa777 There is no problem as long as you feel fine with it.